Tests/MapKitTests/MKMapViewTests.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  MKMapViewTests.swift
+//  SwifterSwift
+//
+//  Created by Hannes Staffler on 24.01.19.
+//  Copyright © 2019 SwifterSwift
+//
+
+import XCTest
+@testable import SwifterSwift
+
+#if canImport(MapKit)
+import MapKit
+import struct CoreLocation.CLLocationCoordinate2D
+
+#if !os(watchOS)
+@available(tvOS 9.2, *)
+final class MKMapViewTests: XCTestCase {
+
+    func testRegister() {
+        let mapView = MKMapView()
+
+        mapView.register(annotationViewWithClass: MKPinAnnotationView.self)
```

Comment:
You aren't actually testing anything in here. You can use the same test as in the other function:
```Swift
let annotationView = mapView.dequeueReusableAnnotationView(withClass: MKPinAnnotationView.self)
XCTAssertNotNil(annotationView)
```

Reply:
ok, updated :)